java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/SessionJsonImpl.java

```diff
@@ -12,6 +12,13 @@
 
   private final EnvelopeEncryption<D> envelopeEncryption;
 
+  /**
+   * Creates a new {@code SessionJsonImpl} instance using the provided {@link EnvelopeEncryption} object. An
+   * implementation of {@link Session} that encrypts a payload of type {@link org.json.JSONObject}.
+   *
+   * @param envelopeEncryption An implementation of
+   *                           {@link com.godaddy.asherah.appencryption.envelope.EnvelopeEncryptionJsonImpl}.
+   */
```

Comment:
this could be any implementation. Not necessary EnvelopeEncryptionJsonImpl. Need to reword it a little